### PR TITLE
Update test-utils website pages

### PR DESCRIPTION
**Pull request by jianhuang01**
#### Background
Test-utils website pages are outdated

#### Change List
- Update test-utils docs

### CODE REVIEW

File: website/contents/pages.js [hunk 1]
```diff
@@ -481,7 +481,7 @@ export const docPages = generatePath([
       },
       {
         name: 'TestRenderer',
-        content: getDocUrl('api-reference/test-utils/test-renderer.md')
+        content: getDocUrl('api-reference/test-utils/renderer-test.md')
```
Comment (Pessimistress) [inline website/contents/pages.js]: If the class is named TestRenderer let's name the file `test-renderer.md`

**Comment by Pessimistress** [comment]
Can you do a search for `deck.gl-test-utils` in the entire docs directory? I believe there are more mentions.

File: website/contents/pages.js (hunk 1 above)
Comment (jianhuang01) [reply website/contents/pages.js]: The class is named as RenderTest

**Comment by jianhuang01** [comment]
@Pessimistress Thanks for reminding, it is done.

File: website/contents/pages.js (hunk 1 above)
Comment (Pessimistress) [reply website/contents/pages.js]: Then you should update the name of this page...

**Pull request merged**